src/Bicep.Core/Analyzers/Linter/LinterRuleBase.cs

```diff
@@ -34,10 +34,10 @@ public abstract class LinterRuleBase : IBicepAnalyzerRule
 
         public string Code { get; }
         public readonly string RuleConfigSection = $"{LinterAnalyzer.SettingsRoot}:{LinterAnalyzer.AnalyzerName}:rules";
-        public Diagnostics.DiagnosticLevel DiagnosticLevel { get; private set; }
+        public DiagnosticLevel DiagnosticLevel { get; private set; }
         public string Description { get; }
-        public string DocumentationUri { get; }
-        public Diagnostics.DiagnosticLabel? DiagnosticLabel { get; }
+        public string Uri { get; }
```

Comment:
Can/should we use the `Uri` type (rather than `string`)?

Reply:
I've twisted around over that. It means we allocate the string and the URI even if it's not used. I'm can go either way.

Reply:
changing to Uri everywhere